tools/DevicesApiTester/Commands/Pwm/DimLed.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Pwm;
+using System.Threading.Tasks;
+using CommandLine;
+using DeviceApiTester.Infrastructure;
+
+namespace DeviceApiTester.Commands.Pwm
+{
+    [Verb("pwm-dim-led", HelpText = "Dims an LED connected to a specified PWM pin for 3 seconds")]
+    public class DimLed : PwmCommand, ICommandVerbAsync
+    {
+        /// <summary>Executes the command asynchronously.</summary>
+        /// <returns>The command's exit code.</returns>
+        /// <remarks>
+        ///     NOTE: This test app uses the base class's <see cref="CreatePwmController"/> method to create a device.<br/>
+        ///     Real-world usage would simply create an instance of <see cref="PwmController"/>:
+        ///     <code>using (var pwm = new PwmController())</code>
+        /// </remarks>
+        public async Task<int> ExecuteAsync()
+        {
+            Console.WriteLine($"LedPin={LedPin}, DutyCycle={DutyCycle}, Frequency={Frequency}, Driver={Driver}");

```

Comment:
`LedPin` might be a little misleading. Even though it is one pin, when you are in PWM you are talking about channels. It may be important to change this, since for example, in a raspberry pi the pin 18, is the pwm channel 0, so the way you have it I wouldn't be super sure if I should pass in a 0 or an 18.

Reply:
I've changed the parameter to "channel" to be more clear